packages/schema-generator/API.md

```diff
@@ -0,0 +1,37 @@
+## API Report File for "@aws-amplify/schema-generator"
+
+> Do not edit this file. It is a report generated by [API Extractor](https://api-extractor.com/).
+
+```ts
+
+// @public
+export const parseDatabaseUrl: (databaseUrl: string) => SQLDataSourceConfig;

```

Comment:
I don't think this should be exported from the package?

Reply:
Removed from public api.